chrome-extension/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "@extension/tsconfig/app",
   "compilerOptions": {
-    "baseUrl": "./",
+    "baseUrl": ".",
```

Comment:
Was this change necessary?

Reply:
No, but i want to do it, to have this config like all others